hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNode.java

```diff
@@ -2509,6 +2518,43 @@ String reconfigureIPCBackoffEnabled(String newVal) {
     return Boolean.toString(clientBackoffEnabled);
   }
 
+  String reconfigureLogSlowRPC(String property, String newVal) throws ReconfigurationException {
+    String result = null;
+    try {
+      if (property.equals(IPC_SERVER_LOG_SLOW_RPC)) {
```

Comment:
I assume this is needed. But why did we NOT have this code for `IPC_SERVER_LOG_SLOW_RPC` before?

Reply:
yes, this is needed